### ISSUE TITLE
[Accessibility]A11y_VSCode_UserSnippets_ColorContrast:Color contrast ratio is less than 4.5:1 for the placeholder and the list items present on the screen.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52044]
Environment   Details:
VSCode Version : 1.24.1     
OS Version : Win10     

Additional Details:     
MAS Violated: MAS1.4.3     

Repro Steps:     
1)Launch VS Code.     
2)Navigate to Settings button present on the left bottom corner of the screen and Select User Snippets.
3)Open Color contrast analyzer and check the ratio.     

Actual:     
Color contrast ratio is less than 4.5:1 for the placeholder text and the list items present on the screen.  Same issue has been observed with many of the color themes. 

Expected:     
Color Contrast ratio should not be less than 4.5:1. 

Recommendations:     
Refer below link which is repository of bug fixes code snippets:       https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx    

User Impact:     
The user will not be able to read the texts.     

MAS Reference:     
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

Attachment for reference: 
![a11y vscode problems colorcontrast](https://user-images.githubusercontent.com/39204909/41469854-71289626-70cc-11e8-9e3c-170718a80f1e.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by RMacfarlane** [comment]
The control that opens here is opened in many different scenarios. Since the issue is with the control's colors, I'm going to close all but one issue that involve opening the same control.

https://github.com/Microsoft/vscode/issues/51831
